### PR TITLE
psp-2167 prevent leading zeros from being stripped from pid search.

### PR DESCRIPTION
**Pull request by devinleighsmith**

### CODE REVIEW

File: backend/dal/Migrations/20211110022507_v1.0.14.2.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```
Comment (FuriousLlama) [inline backend/dal/Migrations/20211110022507_v1.0.14.2.cs:1]: Are these related to the bug? I am guessing the migrations are being piggybacked.

**Review by FuriousLlama — APPROVED** [review]

File: backend/dal/Migrations/20211110022507_v1.0.14.2.cs (hunk 1 above)
Comment (devinleighsmith) [reply backend/dal/Migrations/20211110022507_v1.0.14.2.cs:1]: This is related to the bug yes, the newer view provides the pid in the expected format.

Comment (devinleighsmith) [reply backend/dal/Migrations/20211110022507_v1.0.14.2.cs:1]: Previously the view only exposed the pid as an int, and there isn't a lot you can do in cql to get around that.
